.eslintrc.yml

```diff
@@ -0,0 +1,20 @@
+extends: airbnb-base
+parser: babel-eslint
+env:
+  browser: true
+  node: true
+
+globals:
+  atom: true
+
+settings:
+  import/core-modules: [ atom ]
+
+rules:
+  no-console: [0]
+  no-use-before-define: [0]
+  no-param-reassign: [0]
+  no-underscore-dangle: [0]
+  no-useless-escape: [0]
```

Comment:
Otherwise it would complain about [this line](https://github.com/lgeiger/hydrogen/blob/43e8ae80732fab70a8c37669626d0bdb8009518e/lib/code-manager.js#L208)